source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -196,14 +196,19 @@ class NiDAQmxDriverApiTests : public Test {
     return stub()->GetErrorString(&context, request, &response);
   }
 
-  ::grpc::Status start_task(StartTaskResponse& response = ThrowawayResponse<StartTaskResponse>::response())
+  ::grpc::Status start_task(const ::nidevice_grpc::Session& session, StartTaskResponse& response)
```

Comment:
Note: "Watchdog" is implemented as a separate task. So this PR expands the test helpers to support multiple tasks/sessions.